src/requests/mod.rs

```diff
@@ -18,3 +18,13 @@ pub trait Request {
     /// Asynchronously sends this request to Telegram and returns the result.
     async fn send(&self) -> ResponseResult<Self::Output>;
 }
+
+/// Designates an API request.
+#[async_trait::async_trait]
+pub trait RequestFile {
```

Comment:
Maybe something like `RequestWithFile`? Because `RequestFile` seems like we're requesting a file.

Also, the copy-pasted documentation from `Request` should be updated.